PollBuddy-Server/frontend/src/pages/RegisterWithPollBuddy/RegisterWithPollBuddy.js

```diff
@@ -76,8 +76,10 @@ class RegisterWithPollBuddy extends Component {
             To create an account, fill in the text boxes, then press submit.
           </p>
           <MDBContainer className="form-group">
-            <label htmlFor="nameText">Name:</label>
-            <input placeholder="SIS Man" className="form-control textBox" id="nameText" 
+            <label htmlFor="firstnameText">Name:</label>
```

Comment:
You'll want to put the labels and inputs together I believe. Label1 Input1 Label2 Input2.